### PR TITLE
doc: Update the bt_rpc doc with sysbuild

### PR DESCRIPTION
**Pull request by dchat-nordic**
Update the bt_rpc doc with sysbuild options.
Promote the ipc_radio application.

Jira: NCSDK-28519

### CODE REVIEW

File: doc/nrf/libraries/bluetooth_services/rpc.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ The Bluetooth Low Energy RPC is a solution that consists of the following compon
 
   * Bluetooth RPC Client and common software libraries that serialize the Bluetooth Host API and enable RPC communication.
     These libraries need to be part of the user application.
-  * :ref:`ble_rpc_host` sample that includes the full configured Bluetooth Low Energy stack, Bluetooth RPC Host, and common libraries that enable communication with the Bluetooth RPC Client.
+  * :ref:`ipc_radio` application that includes the full configuaration for Bluetooth Low Energy stack, Bluetooth RPC Host, and common libraries that enable communication with the Bluetooth RPC Client.
```
Comment (grochu) [inline doc/nrf/libraries/bluetooth_services/rpc.rst:26]:
```suggestion
  * :ref:`ipc_radio` application that includes the full configuration of the Bluetooth Low Energy stack, Bluetooth RPC Host and common libraries that enable communication with the Bluetooth RPC Client.
```

File: doc/nrf/libraries/bluetooth_services/rpc.rst [hunk 2]
```diff
@@ -83,53 +83,49 @@ Set the following options in the same way for the :ref:`ble_rpc_host` or :ref:`i
   * :kconfig:option:`CONFIG_BT_DEVICE_NAME`
   * :kconfig:option:`CONFIG_NRF_RPC_CBKPROXY_OUT_SLOTS` on one core must be equal to :kconfig:option:`CONFIG_NRF_RPC_CBKPROXY_IN_SLOTS` on the other.
 
-To keep all the above configuration options in sync, create an overlay file that is shared between the application and network core.
+To keep all the above configuration options in sync, create a snipptet with an overlay file that is shared between the application and network core.
```
Comment (grochu) [inline doc/nrf/libraries/bluetooth_services/rpc.rst:86]:
```suggestion
To keep all the above configuration options in sync, you can create a snippet with an overlay file that is shared between the application and network core.
```
Also consider linking to sysbuild snippet documentation: https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/build/snippets/index.html.

File: doc/nrf/libraries/bluetooth_services/rpc.rst [hunk 3]
```diff
@@ -83,53 +83,49 @@ Set the following options in the same way for the :ref:`ble_rpc_host` or :ref:`i
...
 Then, you can invoke build command like this:
 
 .. parsed-literal::
    :class: highlight
 
-   west build -b *board_target* -- -DEXTRA_CONF_FILE=my_overlay_file.conf
+   west build -b *board_target* -Smy_snippet
```
Comment (grochu) [inline doc/nrf/libraries/bluetooth_services/rpc.rst:92]:
```suggestion
   west build -b *board_target* -S my_snippet
```

File: doc/nrf/libraries/bluetooth_services/rpc.rst [hunk 4]
```diff
@@ -83,53 +83,49 @@ Set the following options in the same way for the :ref:`ble_rpc_host` or :ref:`i
...
-  * :kconfig:option:`CONFIG_BT_RPC_CLIENT`
-  * :kconfig:option:`CONFIG_BT_RPC_HOST`
-  * :kconfig:option:`CONFIG_BT_RPC_STACK`
-  * :kconfig:option:`CONFIG_BT_RPC_INITIALIZE_NRF_RPC`
-  * :kconfig:option:`CONFIG_BT_RPC_GATT_SRV_MAX`
-  * :kconfig:option:`CONFIG_BT_RPC_GATT_BUFFER_SIZE`
-  * :kconfig:option:`CONFIG_BT_RPC_INTERNAL_FUNCTIONS`
-  * :kconfig:option:`CONFIG_NRF_RPC_CBKPROXY_OUT_SLOTS`
-  * :kconfig:option:`CONFIG_NRF_RPC_CBKPROXY_IN_SLOTS`
+   west build -b nrf5340dk/nrf5340/cpuapp -Snordic-bt-rpc -- -DSB_CONFIG_NETCORE_IPC_RADIO=y -DSB_CONFIG_NETCORE_IPC_RADIO_BT_RPC=y
```
Comment (grochu) [inline doc/nrf/libraries/bluetooth_services/rpc.rst:105]:
```suggestion
   west build -b board_name -S nordic-bt-rpc -- -DSB_CONFIG_NETCORE_IPC_RADIO=y -DSB_CONFIG_NETCORE_IPC_RADIO_BT_RPC=y
```

File: doc/nrf/libraries/bluetooth_services/rpc.rst [hunk 5]
```diff
@@ -83,53 +83,49 @@ Set the following options in the same way for the :ref:`ble_rpc_host` or :ref:`i
...
 
-For more details, see the Kconfig option description.
+.. note::
+   The samples which support the Bluetooth Low Energy RPC use the :makevar:`FILE_SUFFIX` along with :makevar:`SNIPPET` to adjust the network/radio core selection and configuration.
```
Comment (grochu) [inline doc/nrf/libraries/bluetooth_services/rpc.rst:108]: Suggestion:
```suggestion
   The samples which support the Bluetooth Low Energy RPC use the :makevar:`FILE_SUFFIX` along with :makevar:`SNIPPET` to adjust the selection and configuration of the network/radio core firmware.
```

File: doc/nrf/libraries/bluetooth_services/rpc.rst [hunk 6]
```diff
@@ -83,53 +83,49 @@ Set the following options in the same way for the :ref:`ble_rpc_host` or :ref:`i
...
+
+* :ref:`central_uart`
+* :ref:`peripheral_hids_mouse`
+* :ref:`peripheral_uart`
 
-The :ref:`ble_rpc_host` sample exposes the Bluetooth LE stack functionality that runs on an MCU with radio (for example, the nRF5340 network core) to another CPU using the :ref:`nrfxlib:nrf_rpc`.
-When building samples for the application core, enable the :kconfig:option:`CONFIG_BT_RPC_STACK` Kconfig option to run the Bluetooth LE stack on the network core.
-This option builds :ref:`ble_rpc_host` automatically as a child image.
-For more details, see :ref:`ug_nrf5340_building`.
+The :ref:`ipc_radio` sample, is a configurable sample, which exposes the Bluetooth LE stack functionality that runs on an MCU with radio (for example, the nRF5340 network core) to another CPU using the :ref:`nrfxlib:nrf_rpc`.
```
Comment (grochu) [inline doc/nrf/libraries/bluetooth_services/rpc.rst:124]:
```suggestion
The :ref:`ipc_radio` sample, is a configurable application, that exposes the Bluetooth LE stack functionality running on an MCU with radio (for example, the nRF5340 network core) to another CPU through the :ref:`nrfxlib:nrf_rpc`.
```

File: doc/nrf/libraries/bluetooth_services/rpc.rst [hunk 7]
```diff
@@ -83,53 +83,49 @@ Set the following options in the same way for the :ref:`ble_rpc_host` or :ref:`i
...
+When using the :ref:`ipc_radio` as Bluetooth Low Energy over RPC reconfigure it by setting the ``SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC=y`` Sysbuild Kconfig option to run the Bluetooth LE stack on the network core.
```
Comment (grochu) [inline doc/nrf/libraries/bluetooth_services/rpc.rst:125]:
```suggestion
When using the :ref:`ipc_radio` as Bluetooth Low Energy over RPC reconfigure it by setting the ``SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC=y`` Sysbuild Kconfig option to run the whole Bluetooth LE stack on the network core.
```

**Review by grochu — CHANGES_REQUESTED** [review]
There are small TYPO and my other comments are suggestions for improvement.

File: doc/nrf/libraries/bluetooth_services/rpc.rst (hunk 7 above)
Comment (divipillai) [inline doc/nrf/libraries/bluetooth_services/rpc.rst:125]:
```suggestion
When using the :ref:`ipc_radio` as Bluetooth Low Energy over RPC, reconfigure it by setting the ``SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC`` sysbuild Kconfig option to ``y`` to run the whole Bluetooth LE stack on the network core.
```

File: doc/nrf/libraries/bluetooth_services/rpc.rst [hunk 8]
```diff
@@ -83,53 +83,49 @@ Set the following options in the same way for the :ref:`ble_rpc_host` or :ref:`i
...
+For more details, see :ref:`ipc_radio_config`.
```
Comment (divipillai) [inline doc/nrf/libraries/bluetooth_services/rpc.rst:126]:
```suggestion
For more details, see the :ref:`Configuration <ipc_radio_config>` section of the IPC radio firmware application guide.
```

**Review by divipillai — APPROVED** [review]

**Review by maje-emb — APPROVED** [review]

**Review by grochu — APPROVED** [review]
Looks good

**Pull request merged**